File: src/main/java/com/distribution/modules/memeber/dao/WithdrawalInfoDao.java
package com.distribution.modules.memeber.dao;

import com.distribution.modules.memeber.entity.WithdrawalInfo;
import com.distribution.modules.sys.dao.BaseDao;
import org.apache.ibatis.annotations.Mapper;

import java.util.List;

@Mapper
public interface WithdrawalInfoDao extends BaseDao<WithdrawalInfo> {

<<<<<<< HEAD
    List<WithdrawalInfo> findAmounts(String withdrawMobile);
=======
    List<WithdrawalInfo> findByAliPayAccount(String withdrawMobile);
>>>>>>> origin/develop
}

File: src/main/kotlin/com/distribution/modules/memeber/dao/WithdrawalInfoDao.kt
package com.distribution.modules.memeber.dao

import com.distribution.modules.memeber.entity.WithdrawalInfo
import com.distribution.modules.sys.dao.BaseDao
import org.apache.ibatis.annotations.Mapper

@Mapper
interface WithdrawalInfoDao : BaseDao<WithdrawalInfo> {

    fun findByAliPayAccount(withdrawMobile: String): List<WithdrawalInfo>
}
